origin-bridge/logic/attestation_service.py

```diff
@@ -81,6 +81,10 @@ def send_phone_verification(country_calling_code, phone, method, locale):
             # If a locale is not set Twilio will use a sensible default based on
             # the country of the telephone number
             params['locale'] = locale
+        elif country_calling_code == '91':
+            # Locale not provided, and calling India, so we want to set English
+            # instead of Hindi, which is Twilio's default there (#1124)
```

Comment:
I haven't seen this pattern here of referencing issues, not sure if it's acceptable.

Reply:
It's something I think we should do more of 👍